custom/panel_templates/Default/members/member_lists.tpl

```diff
@@ -0,0 +1,116 @@
+{include file='header.tpl'}
+
+<body id="page-top">
+
+<!-- Wrapper -->
+<div id="wrapper">
+
+    <!-- Sidebar -->
+    {include file='sidebar.tpl'}
+
+    <!-- Content Wrapper -->
+    <div id="content-wrapper" class="d-flex flex-column">
+
+        <!-- Main content -->
+        <div id="content">
+
+            <!-- Topbar -->
+            {include file='navbar.tpl'}
+
+            <!-- Begin Page Content -->
+            <div class="container-fluid">
+
+                <!-- Page Heading -->
+                <div class="d-sm-flex align-items-center justify-content-between mb-4">
+                    <h1 class="h3 mb-0 text-gray-800">{$MEMBER_LISTS}</h1>
+                    <ol class="breadcrumb float-sm-right">
+                        <li class="breadcrumb-item"><a href="{$PANEL_INDEX}">{$DASHBOARD}</a></li>
+                        <li class="breadcrumb-item active">{$MEMBER_LISTS}</li>
+                    </ol>
+                </div>
+
+                <!-- Update Notification -->
+                {include file='includes/update.tpl'}
+
+                <div class="card shadow mb-4">
+                    <div class="card-body">
+                        <!-- Success and Error Alerts -->
+                        {include file='includes/alerts.tpl'}
+
+                        <form id="toggleHideBannedUsers" action="" method="post">
+                            <input type="hidden" name="token" value="{$TOKEN}">
+                            <div class="form-group custom-control custom-switch">
+                                <input type="hidden" name="action" value="toggle_hide_banned_users">
+                                <input id="inputToggleHideBannedUsers" name="hide_banned_users" type="checkbox" class="custom-control-input js-check-change" value="1" {if $HIDE_BANNED_USERS_VALUE eq 1} checked{/if} />
+                                <label for="inputToggleHideBannedUsers" class="custom-control-label">
+                                    {$HIDE_BANNED_USERS}
```

Comment:
Probably not a day one request but maybe we could make this configurable per member list?